core-common/src/main/java/org/glassfish/jersey/message/internal/CommittingOutputStream.java

```diff
@@ -59,7 +59,7 @@
 public final class CommittingOutputStream extends OutputStream {
 
     private static final Logger LOGGER = Logger.getLogger(CommittingOutputStream.class.getName());
-    private static final boolean JDK_21 = JdkVersion.getJdkVersion().getMajor() > 20;
+    private final boolean isVirtualThread = VirtualThreadSupport.isVirtualThread();
```

Comment:
I am just curious, was there any issue with the next line?:
`private static final boolean JDK_21 = JdkVersion.getJdkVersion().getMajor() > 20;`